src/stats/hist.jl

```diff
@@ -180,6 +181,11 @@ function Makie.plot!(plot::Hist)
         x === :values ? :y : x
     end
     # plot the values, not the observables, to be in control of updating
+
+    for key in [:bins, :normalization, :over_background_color, :over_bar_color, :scale_to, :weights]
```

Comment:
Any particular reason why?  Users could still play around with these, no?

Reply:
ah yeah meant to delete these from a copy